staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/queueset.go

```diff
@@ -102,9 +102,9 @@ type queueSet struct {
 	// queues are still draining.
 	queues []*queue
 
-	// virtualTime is the amount of seat-seconds allocated per queue since process startup.
+	// currentR is the amount of seat-seconds allocated per queue since process startup.
 	// This is our generalization of the progress meter named R in the original fair queuing work.
-	virtualTime float64
+	currentR SeatSeconds
```

Comment:
It's not that I'm opposed to this change - in fact I'm 100% supportive.
But mixing different things within a single commit (i.e. renaming of variables, logic etc. makes it much harder to review it...)

Reply:
There is no logic change here.

The only fields whose name change are fields whose type changed, so all the same lines of code get touched.

Reply:
I don't fully agree - most of the lines affected by rename are logs. Separating those would actually make this PR simpler.

Reply:
The logs have to be changed anyway.

Reply:
Most of them are just changing only names - so I don't fully agree.
But I'm not going to push you for changing that - it's more a comment for the future.